Ghidra/RuntimeScripts/Linux/support/launch.sh

```diff
@@ -154,6 +154,8 @@ else
 	exit 1
 fi
 
+VMARG_LIST+=" -XX:+IgnoreUnrecognizedVMOptions -XX:+ShowCodeDetailsInExceptionMessages"
```

Comment:
Do you know about the launch.properties file?

Reply:
Yes. For some reason it didn't click to use it but that should have been obvious.

Reply:
Unfortunately the Ghidra.launch file still needs to be updated.  I really hoped that there was a JVM argument that could take a path to a properties file so they wouldn't have to be defined twice, but I couldn't find anything like that.

Reply:
> Unfortunately the Ghidra.launch file still needs to be updated.  I really hoped that there was a JVM argument that could take a path to a properties file so they wouldn't have to be defined twice, but I couldn't find anything like that.

I've set them in both. I think this should be good now. My only concern is that if a user adds a vmarg and makes a typo they may not realize it is being ignored which could be a headache to debug.